doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -164,8 +164,15 @@ Caveats:
 
 A VNF application network topology is expressed in terms of VMs, vNIC interfaces with vNet access networks, and WAN Networks while the VNF Application VMs require multiple vNICs, VLANs, and host routes configured within the VM’s Kernel.
 
-#### 4.2.3.3. LBaaS v2 compliant Load Balancing
-**Content to be developed**
+#### 4.2.3.3. Octavia v2 API compliant Load Balancing
+Load balancing is needed for automatic scaling, managing availability and changes. [Octavia](https://docs.openstack.org/octavia/latest/reference/introduction.html) is an open-source load balancer for OpenStack, based on HAProxy, and replaces the deprecated (as of OpenStack Queens release) Neutron LBaaS. The Octavia v2 API is a superset of the deprecated Neutron LBaaS v2 API and has a similar CLI for seamless transition. 
+
+As a default Octavia utilizes Amphorae Load Balancer. Amphorae consists of a fleet of VMs, containers or bare metal servers and delivers horizontal scaling by managing and spinning these resources on demand. The reference implementation of the Amphorae image is an Ubuntu virtual machine running HAProxy. 
+
+Octavia depends upon a number of OpenStack services including Nova for spinning up compute resources on demand and their life cycle management; Neutron for connectivity between the compute resources, project environment and external networks; Keystone for authentication; and Glance for storing of the compute resource images.
```

Comment:
Skipping Barbican for TLS management is intentional?

Reply:
@deebhatia Not trying to get into all of the Octavia interactions. But if they are important then could add a sentence.

Reply:
Okay.